cogs3/urls.py

```diff
@@ -66,6 +66,15 @@
         RegisterView.as_view(),
         name='register',
     ),
+    path(
+        'accounts/complete-registration/',
```

Comment:
How does this complete-registration path alter the current user registration process?

Reply:
It doesn't - this route is to facilitate completing registration for those who have had their CustomUser objects created by someone else specifying them as PI. New users go to the existing route, existing users whose profiles are incomplete go to this route.